### DISCUSSION TITLE
Requesting Review for my Pull Request

### DISCUSSION THREAD

**Discussion by Levi0804** [aptos-labs/aptos-developer-discussions#550] [General]
Hi there! I recently created a pull request on the Aptos developer docs to fix an example. Could you let me know if this is a good improvement over the previous one?

Here's the link to the PR: https://github.com/aptos-labs/developer-docs/pull/719

thanks! :D

**Comment by 0xmaayan** [comment]
Hi @Levi0804 thank you for your contribution! The team is looking at it
